packit/dg_robot.py

```diff
@@ -0,0 +1,256 @@
+import os
+import shutil
+import tempfile
+from typing import List
+
+from ogr.services.pagure import PagureService
+from rebasehelper.specfile import SpecFile
+
+from packit.config import Config
+from packit.local_project import LocalProject
+from packit.sync import logger
+from packit.utils import FedPKG
+
+
+# TODO: create a DistGit class
+class DistGitRobot:
+    """
+    A class which interacts with dist-git and pagure-over-dist-git API.
```

Comment:
I am not sure about the difference between `DistGitRobot`, `PackitDistGitRobot` and `Transformator`. We can discuss that at the meeting.

Reply:
DistGitRobot (will turn it into DistGit), hopefully we'll be able to provide it as an API so Michal can use it.